stablehlo/conversions/linalg/transforms/CMakeLists.txt

```diff
@@ -0,0 +1,26 @@
+set(LLVM_TARGET_DEFINITIONS Passes.td)
+mlir_tablegen(Passes.h.inc -gen-pass-decls -name StablehloLinalgTransforms)
+add_public_tablegen_target(StablehloLinalgTransformsPassIncGen)
+
+add_mlir_library(StablehloLinalgTransforms
+  LegalizeToLinalgUtils.cpp
+  StableHloLegalizeToLinalg.cpp
+  StableHloToArith.cpp
+  StableHloToLinalgConvolution.cpp
+  StableHloToLinalgDotProduct.cpp
+  StableHloToLinalgPointwise.cpp
+  StableHloToLinalgRandom.cpp
+  StableHloToLinalgReduce.cpp
```

Comment:
Can we rename all files `StableHlo* -> Stablehlo*` as is the convention in this repo

Reply:
Renamed these, including MapStableHloToScalarOp.h to MapStablehloToScalarOp.h.